edx_proctoring/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import
 
-__version__ = '0.18.2'
+__version__ = '0.18.3'
```

Comment:
Does a patch version bump sound right to people? I feel like I always do semantic versioning wrong.